.github/workflows/swift-toolchain.yml

```diff
@@ -245,11 +250,20 @@ jobs:
           show-progress: false
 
       - uses: compnerd/gha-setup-vsdevenv@main
+        if: inputs.builder_os == 'Windows'
         with:
-          host_arch: amd64
+          host_arch: ${{ inputs.builder_arch }}
           components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
           arch: ${{ matrix.arch }}
 
+      - uses: maxim-lobanov/setup-xcode@v1
```

Comment:
This is not a whitelisted action, I think this will fail. Do we need to add this dependency?

Reply:
Okay, we can use the action, but will need to configure the repo before merging.